src/coreclr/tools/superpmi/superpmi/superpmi.cpp

```diff
@@ -171,7 +181,8 @@ int __cdecl main(int argc, char* argv[])
 #endif
 
     bool   collectThroughput = false;
-    MCList failingToReplayMCL, diffMCL;
+    MCList failingToReplayMCL;
```

Comment:
It will likely be useful to give the failing list the same treatment so that we can have `superpmi.py replay` print the smallest contexts with failures, instead of printing everything. I will leave this for a future PR however.